src/frontend/src/pages/GanttPage/GanttPage.tsx

```diff
@@ -5,22 +5,26 @@
 
 import { Gantt } from './GanttPackage/components/gantt/gantt';
 import { Task, ViewMode } from './GanttPackage/types/public-types';
-import { useState } from 'react';
+import { FC, useEffect, useState } from 'react';
 
 interface GanttPageProps {
-  tasks: Task[];
+  ganttDisplayObjects: Task[];
 }
 
-const GanttPage: React.FC<GanttPageProps> = ({ tasks }) => {
-  const [taskList, setTaskList] = useState<Task[]>(tasks);
+const GanttPage: FC<GanttPageProps> = ({ ganttDisplayObjects }) => {
+  const [ganttDisplayObjectList, setGanttDisplayObjectList] = useState<Task[]>(ganttDisplayObjects);
+
+  useEffect(() => {
```

Comment:
this just means you dont' need a usestate or useeffect at all and can just use the prop as is